modules/i3s/src/lib/parsers/parse-zip/cd-file-header.ts

```diff
@@ -28,15 +42,14 @@ export type ZipCDFileHeader = {
 export const parseZipCDFileHeader = async (
   headerOffset: bigint,
   buffer: FileProvider
-): Promise<ZipCDFileHeader> => {
-  const offsets = {
-    CD_COMPRESSED_SIZE_OFFSET: 20n,
-    CD_UNCOMPRESSED_SIZE_OFFSET: 24n,
-    CD_FILE_NAME_LENGTH_OFFSET: 28n,
-    CD_EXTRA_FIELD_LENGTH_OFFSET: 30n,
-    CD_LOCAL_HEADER_OFFSET_OFFSET: 42n,
-    CD_FILE_NAME_OFFSET: 46n
-  };
+): Promise<ZipCDFileHeader | undefined> => {
```

Comment:
`undefined` is not a good type to return it explicitly. `null` is better
```suggestion
): Promise<ZipCDFileHeader | null> => {
```